lib/nexpose/scan_template.rb

```diff
@@ -176,6 +176,56 @@ def host_threads=(threads)
       host_threads.text = threads.to_s
     end
 
+    # Enable/disable IP stack fingerprinting
+    # @param [Boolean] enable or disable IP stack fingerprinting
+    def enable_ip_stack_fingerprinting(enable)
```

Comment:
I'll change this to the same `foo=(blah)` style used by the others, for consistency's sake.